tests/urls.py

```diff
@@ -17,6 +17,6 @@
 from django.contrib import admin
 
```

Comment:
Do you think we should keep the comments in lines 1-15? I am thinking maybe not.

Reply:
Not a fan of them. We don't keep them in our other projects. I'll eliminate them. With extreme prejudice.